### ISSUE TITLE
Feature request for the 'Trigger Azure DevOps Pipeline' extension: wait for the build/release to complete

### ISSUE THREAD

**Issue by rajbos** [maikvandergaag/msft-extensions#93]
Hi Maik,

It would be great we could wait for the triggered (build or) release to finish. This way, we can create single releases to quickly deploy one part of the system, and create one release to group them all if we need to roll out to e.g. a new system with everything on it. 

The `Trigger Build Task` from [Benj Huser](https://marketplace.visualstudio.com/items?itemName=benjhuser.tfs-extensions-build-tasks) for example enables to wait on a build. I need it for a release :-)

**Comment by harobaz** [comment]
Hi Maik,

Thanks for your work !
I have the same need here, it will be great if your task could wait the build/release to finish !

**Comment by maikvandergaag** [comment]
Sorry for the late response due to some personal stuff I haven't been able to proccess this hope to be able to do this in august

**Comment by rajbos** [comment]
Hi Maik, no worries, nu rush! Take care of yourself!

**Comment by seantaylortrintech** [comment]
Hi Maik. I also have the same request as I have a need to execute pipelines in serial, one right after another and not in parallel. Thank you for this extension.

**Comment by danieleusebio** [comment]
Hi,
Great plugin and for me very useful.
I have the same need here. The downstream pipeline will work as a sub-pipeline(or a function) if the main pipeline capture its variables.
Thanks in advance.

**Comment by danquirk** [comment]
See https://github.com/dmitryserbin/azdev-release-orchestrator for a plugin that does this for release pipelines today

**Comment by KevinWilliamsAlpha** [comment]
Just dropping in another request for this feature!

**Comment by kumarkaran1** [comment]
my request is if you will add this feature please give the option to execute parallel release as well

**Comment by aemurd-n** [comment]
While waiting for this feature, has anyone figured out a different way of doing this, please?

**Comment by sanju0001** [comment]
Added a code to wait for the succeeded status of the Release Pipeline in the existing script :

`                        do{
                            $resultGetRelease = Invoke-RestMethod -Uri $RunEnvUri -Method Get -ContentType "application/json" -Headers $DevOpsHeaders;
                        
                            if($resultGetRelease.status -ne "succeeded"){
                                Start-Sleep -Seconds 5  
                            }
                        Write-Host $resultGetRelease.status
                        }while($resultGetRelease.status -ne "succeeded")    
                        `

**Comment by maikvandergaag** [comment]
@sanju0001 thanks for the script will see if I can add it to the script. I am on holiday now will check next week

**Comment by pbcahill** [comment]
@sanju0001 what are you passing in for the $RunEnvUri variable in your script? How do you know what the Release ID is for the run triggered by the "Trigger Azure DevOps Pipeline" task? In the logs of the trigger task, it does mention the triggered Release ID, but it is not output to an env variable that can be referenced in another task in the pipeline from what I can tell.
